llm_bench/python/utils/model_utils.py

```diff
@@ -49,6 +50,16 @@ def set_default_param_for_ov_config(ov_config):
         ov_config['NUM_STREAMS'] = '1'
 
 
+def add_stateful_model_arguments(parser: argparse.ArgumentParser):
+    parser.add_argument(
+        '--stateful',
```

Comment:
should we name it --make_stateful for backward compatibility?

Reply:
Validation will apply necessary changes, I wouldn't build any backward compatibility story for experimental keys here. It is named stateful for simplicity and alignment with optimum-intel.